docs/components/DashboardExamples.vue

```diff
@@ -0,0 +1,323 @@
+<template>
+    <div class="dashboard-examples">
+        <!-- Carousel Container -->
+        <div class="carousel-container">
+            <button class="nav-button prev" @click="prevImage" :disabled="currentIndex === 0">
+                <ChevronLeft />
+            </button>
+
+            <div class="carousel" ref="carousel">
+                <div
+                    class="carousel-track"
+                    :style="{ transform: `translateX(-${currentIndex * 100}%)` }"
+                >
+                    <div
+                        v-for="(image, index) in images"
+                        :key="index"
+                        class="carousel-slide"
+                    >
+                        <img :src="image.url" :alt="image.alt" @click="openFullscreen(image)">
+                        <label class="carousel-slide-label" @click.stop>
+                            {{ image.alt }}
+                        </label>
+                    </div>
+                </div>
+                <a v-if="images[currentIndex].link" class="learn-more VPButton medium alt" :href="images[currentIndex].link" target="_blank">
+                    <ExternalLink />
+                </a>
+            </div>
+
+            <button class="nav-button next" @click="nextImage" :disabled="currentIndex === images.length - 1">
+                <ChevronRight />
+            </button>
+        </div>
+
+        <!-- Thumbnail Navigation -->
+        <div class="thumbnails">
+            <div
+                v-for="(image, index) in images"
+                :key="index"
+                class="thumbnail"
+                :class="{ active: currentIndex === index }"
+                @click="goToImage(index)"
+            >
+                <img :src="image.url" :alt="image.alt">
+            </div>
+        </div>
+
+        <!-- Fullscreen Modal -->
+        <div v-if="fullscreenImage" class="fullscreen-modal" @click="closeFullscreen">
+            <img :src="fullscreenImage.url" :alt="fullscreenImage.alt">
+            <button class="close-button" @click="closeFullscreen">&times;</button>
+        </div>
+    </div>
+</template>
+
+<script>
+import ChevronLeft from './icons/ChevronLeft.vue'
+import ChevronRight from './icons/ChevronRight.vue'
+import ExternalLink from './icons/ExternalLink.vue'
+
+export default {
+    name: 'DashboardExamples',
+    components: {
+        ChevronLeft,
+        ChevronRight,
+        ExternalLink
+    },
+    data () {
+        return {
+            currentIndex: 0,
+            fullscreenImage: null,
+            images: [
+                {
+                    url: '/images/dashboard-examples/oee-dashboard.png',
+                    alt: 'OEE Performance Dashboard',
+                    link: 'https://flowfuse.com/blog/2025/04/building-oee-dashboard-with-flowfuse-2/'
+                },
```

Comment:
```suggestion
                /* {
                    url: '/images/dashboard-examples/oee-dashboard.png',
                    alt: 'OEE Performance Dashboard',
                    link: 'https://flowfuse.com/blog/2025/04/building-oee-dashboard-with-flowfuse-2/'
                }, */
```

Consider disabling this entry until article is released

Reply:
Never mind - article is being published as I ~~speak~~ write